scss_test.scss

```diff
@@ -0,0 +1,4 @@
+.parent {
+  display: inline-block;
+  background: #fff;
```

Comment:
Line should be indented 4 spaces, but was indented 2 spaces<br>Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.